nix/fastlane/gemset.nix

```diff
@@ -196,15 +196,15 @@
     version = "2.1.5";
   };
   fastlane = {
-    dependencies = ["CFPropertyList" "addressable" "babosa" "colored" "commander-fastlane" "dotenv" "emoji_regex" "excon" "faraday" "faraday-cookie_jar" "faraday_middleware" "fastimage" "gh_inspector" "google-api-client" "google-cloud-storage" "highline" "json" "mini_magick" "multi_json" "multi_xml" "multipart-post" "plist" "public_suffix" "rubyzip" "security" "simctl" "slack-notifier" "terminal-notifier" "terminal-table" "tty-screen" "tty-spinner" "word_wrap" "xcodeproj" "xcpretty" "xcpretty-travis-formatter"];
+    dependencies = ["CFPropertyList" "addressable" "babosa" "colored" "commander-fastlane" "dotenv" "emoji_regex" "excon" "faraday" "faraday-cookie_jar" "faraday_middleware" "fastimage" "gh_inspector" "google-api-client" "google-cloud-storage" "highline" "json" "jwt" "mini_magick" "multi_xml" "multipart-post" "plist" "public_suffix" "rubyzip" "security" "simctl" "slack-notifier" "terminal-notifier" "terminal-table" "tty-screen" "tty-spinner" "word_wrap" "xcodeproj" "xcpretty" "xcpretty-travis-formatter"];
     groups = ["default"];
     platforms = [];
     source = {
       remotes = ["https://rubygems.org"];
       sha256 = "1rz1v4zdxrp09s1l7ck32gszr8a3wg866a2l9gy6hj0a1b8854cc";
```

Comment:
This should have changed to `0n9jf7h262dqsc5wbp3jv2l2mys8y8if5ipyq7hjm8plgp8w5is5` 🤔 https://circleci.com/gh/expo/expo/52463

![Zrzut ekranu 2019-06-3 o 12 59 52](https://user-images.githubusercontent.com/1151041/58797142-7e157180-85ff-11e9-9015-2cc5f276072d.png)